Dockerfile

```diff
@@ -1,25 +1,30 @@
 FROM node:16-alpine AS BUILD_IMAGE
 
-RUN apk update && apk add --no-cache curl bash tini git docker-cli docker-compose grep make python3 g++
+# RUN apk update && apk add --no-cache curl bash tini git docker-cli docker-compose grep make python3 g++
```

Comment:
Have you test the local build docker image, seems it shouldn't work if missing `docker-cli`

Reply:
yes, it's tested. there are two phases building two differrent images. the `BUILD_IMAGE` is only for building node modules. and the second image will install system packages including `docker-cli` which will actually take effect when it's working.